.github/workflows/docker.yml

```diff
@@ -0,0 +1,34 @@
+name: Build and publish Docker images
+
+on:
+  push:
+    branches:
+      - '1.x'
+
+jobs:
+  testrunner:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v2
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v2
+
+      - name: Login to the GitHub Container Registry
+        uses: docker/login-action@v2
+        with:
+          registry: ghcr.io
+          username: publisher
+          password: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Build and push
+        uses: docker/build-push-action@v3
+        with:
+          context: docker
+          file: docker/Dockerfile.buster
+          platforms: "linux/amd64,linux/arm64"
+          push: true
+          tags: datadog/dd-trace-py/testrunner:latest
```

Comment:
should we tag it with the hash as well so we can check out at various points?

Reply:
does it matter much? the commit/hash from this repo doesn't necessarily mean something changes in the dockerfile/image?

Reply:
I think putting the sha on it makes sense so we can easily access a specific version.